src/widgets/topicFrequencyGraph.js

```diff
@@ -8,7 +8,7 @@ import {
 import { REPORT_STATUSES } from '../constants';
 
 export default async function topicFrequencyGraph(scopes) {
-  const topicsAndParticipants = await ActivityReport.findAll({
+  const topicsAndParticipants = await ActivityReport.scope().findAll({
```

Comment:
Not sure that ```.scope()``` is doing anything here or in the other places you are using it. I compared the SQL generated with and without and I saw close to no changes. Looks like any performance improvements must come from bringing in limited attributes.

I'd recommend double checking and removing "scope" if possible so we can reduce confusion as to what it is doing next time someone has to open this file.

Feels like a real performance improvement would be: reduce the number of DB queries, since we are doing 3 here

Reply:
other than that, looks good, saw a reduction of several seconds, which is excellent

Reply:
I only say that because in the past it's been the time of connection from sequelize to postgres that was the source of slowdown. None of these queries seem particularly onerous so my assumption was that that was the case here

Reply:
@GarrettEHill did a rewrite on this running all queries in parallel along with some postgres magic to get all topics in a single result.

I think this is good to go assuming all tests pass.

Reply:
ultimately I'm seeing extremely varied results here from the same query, anything from around 2 seconds to around 10. Still I'm presuming this is a performance enhancement of some kind so I'll approve